packages/alert/src/Alert.tsx

```diff
@@ -0,0 +1,91 @@
+import CheckIcon from '@littlespoon/icons/lib/CheckIcon'
+import ExclamationIcon from '@littlespoon/icons/lib/ExclamationIcon'
+import colors from '@littlespoon/theme/src/colors'
```

Comment:
Replace `src` with `lib`:

```suggestion
import colors from '@littlespoon/theme/lib/colors'
```